code/modules/mob/living/carbon/human/life/handle_organs.dm

```diff
@@ -11,57 +10,56 @@
 		if(!(status_flags & GODMODE)) //godmode doesn't work as intended anyway
 			reagents.metabolize(src, overdosable, L ? FALSE : TRUE)
 
-	if(!(species.species_flags & IS_SYNTHETIC))
+	if(species.species_flags & IS_SYNTHETIC)
+		nutrition = 350 //synthetics are never hungry
+
+	else
+
 		//Nutrition decrease
 		if(nutrition > 0 && stat != 2)
 			nutrition = max (0, nutrition - HUNGER_FACTOR)
 
 		if(nutrition > 450)
-			if(overeatduration < 600) //Capped so people don't take forever to unfat
+			if(overeatduration < 600)	//Capped so people don't take forever to unfat
```

Comment:
whitespace change

Reply:
Want me to revert it? Linter was complaining so I tried some things.